src/test/java/com/exasol/adapter/document/files/IntegrationTestSetup.java

```diff
@@ -0,0 +1,193 @@
+                .language(UdfScript.Language.JAVA).inputType(UdfScript.InputType.SET)
+                .parameter(PARAMETER_DOCUMENT_FETCHER, "VARCHAR(2000000)")
+                .parameter(PARAMETER_SCHEMA_MAPPING_REQUEST, "VARCHAR(2000000)")
+                .parameter(PARAMETER_CONNECTION_NAME, "VARCHAR(500)").emits()
+                .bucketFsContent(UdfEntryPoint.class.getName(), "/buckets/bfsdefault/default/" + ADAPTER_JAR).build();
+    }
+
+    private ConnectionDefinition createConnectionDefinition() {
+        final JsonObjectBuilder configJson = getConnectionConfig();
+        return createConnectionDefinition(configJson);
+    }
+
+    private Optional<String> getHostOverride() {
+        return this.adlsTestSetup.getHostOverride().map(address -> this.exasolTestSetup
+                .makeTcpServiceAccessibleFromDatabase(ServiceAddress.parse(address)).toString());
+    }
+
+    public JsonObjectBuilder getConnectionConfig() {
+        final JsonObjectBuilder objectBuilder = Json.createObjectBuilder();
+        return objectBuilder//
+                .add("adlsContainerName", this.adlsContainer.getFileSystemName())//
+                .add("adlsStorageAccountName", this.adlsTestSetup.getStorageAccountName())//
+                .add("adlsStorageAccountKey", this.adlsTestSetup.getStorageAccountKey());//
+    }
+
+    public ConnectionDefinition createConnectionDefinition(final JsonObjectBuilder details) {
+        String json = toJson(details.build());
+        return this.exasolObjectFactory.createConnectionDefinition("ADLS_CONNECTION_" + System.currentTimeMillis(), "",
+                "", json);
+    }
+
+    private String toJson(final JsonObject configJson) {
+        try (final ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+             final JsonWriter writer = Json.createWriter(outputStream)) {
+            writer.write(configJson);
+            return outputStream.toString(StandardCharsets.UTF_8);
+        } catch (final IOException exception) {
+            throw new UncheckedIOException("Failed to serialize connection settings", exception);
+        }
+    }
+
+    private AdapterScript createAdapterScript(final ExasolSchema adapterSchema)
+            throws BucketAccessException, TimeoutException, FileNotFoundException {
+        this.bucket.uploadFile(Path.of("target", ADAPTER_JAR), ADAPTER_JAR);
+        return adapterSchema.createAdapterScriptBuilder("FILES_ADAPTER")
+                .bucketFsContent("com.exasol.adapter.RequestDispatcher", "/buckets/bfsdefault/default/" + ADAPTER_JAR)
+                .language(AdapterScript.Language.JAVA).build();
+    }
+
+    @Override
+    public void close() throws Exception {
+        try {
+            this.udfTestSetup.close();
+            this.exasolStatement.close();
+            this.exasolConnection.close();
+            this.exasolTestSetup.close();
+        } catch (final SQLException exception) {
+            // at least we tried to close it
+        }
```

Comment:
```suggestion
        this.udfTestSetup.close();
        this.exasolStatement.close();
        this.exasolConnection.close();
        this.exasolTestSetup.close();
```
No need to catch the exception. Better fail instead of hiding the error.